test/msw/mockAPI.js

```diff
@@ -0,0 +1,22 @@
+import { rest } from "./server";
+
+const coingecko = [
+  rest.get(
+    "https://api.coingecko.com/api/v3/simple/token_price/ethereum",
```

Comment:
@richardmands Let's put hardcoded URLs as a constants variables somewhere.

Reply:
Done, see new constant COIN_GECKO_ETHEREUM. Let me know if you're happy with the name.